test/TemplateRenderLiquidTest.js

```diff
@@ -117,17 +123,28 @@ test("Liquid Render Relative (parent dir) Include", async (t) => {
       liquidOptions: {
         dynamicPartials: false,
       },
+    },
+    {
+      dir: {
+        includes: "relative-liquid",
+      },
     }
   );
 
   let fn = await tr.getCompiledTemplate("<p>{% include ../dir/included %}</p>");
   t.is(await fn(), "<p>TIME IS RELATIVE.</p>");
 });
 
-test.skip("Liquid Render Relative (relative include should ignore _includes dir) Include", async (t) => {
+test("Liquid Render Relative (relative include should ignore _includes dir) Include", async (t) => {
   let tr = getNewTemplateRender(
     "./test/stubs/does_not_exist_and_thats_ok.liquid",
-    "./test/stubs/"
+    "./test/stubs/",
+    {},
+    {
+      dir: {
+        includes: ".",
```

Comment:
filepath relative to current file has higher priority than filepath relative to root directory, but still it need to be within `includes`